src/main/java/io/ncbpfluffybear/fluffymachines/machines/AutoAncientAltar.java

```diff
@@ -335,14 +336,21 @@ private void craftIfValid(Block block, boolean craftOnce) {
         if (Constants.isSoulJarsInstalled && sfCatalyst != null
             && sfCatalyst.getId().startsWith("FILLED") && sfCatalyst.getId().endsWith("SOUL_JAR")) {
 
-            SlimefunItem spawnerItem = SlimefunItem.getById(sfCatalyst.getId().replace("FILLED_", "").replace(
-                "_SOUL_JAR", "_BROKEN_SPAWNER"));
+//            SlimefunItem spawnerItem = SlimefunItem.getById(sfCatalyst.getId().replace("FILLED_", "").replace(
+//            "_SOUL_JAR", "_BROKEN_SPAWNER"));
+            EntityType entityType = EntityType.valueOf(sfCatalyst.getId()
```

Comment:
Add a null check to handle potential issues

Reply:
I'd prefer if you did `if (entityType == null) { return; }` (Decreases number of indentations, more concise to me)

Reply:
the `valueOf` method of enum will throw `IllegalArgumentException` if the given string is invalid, and will never be null, this is why I catch it.